var/spack/repos/builtin/packages/exodusii/package.py

```diff
@@ -46,34 +46,43 @@ class Exodusii(Package):
     homepage = "https://github.com/gsjaardema/seacas"
     url      = "https://github.com/gsjaardema/seacas/archive/master.zip"
 
-    version('2016-02-08',
-            git='https://github.com/gsjaardema/seacas.git', commit='dcf3529')
+    version('2016-08-09',
+            git='https://github.com/gsjaardema/seacas.git', commit='2ffeb1b')
 
-    depends_on('cmake@2.8.7:', type='build')
-    depends_on('hdf5~shared~mpi')
-    depends_on('netcdf~mpi')
+    variant('mpi', default=False, description='Enables MPI parallelism.')
 
-    patch('exodus-cmake.patch')
+    depends_on('cmake@2.8.11:', type='build')
+    depends_on('mpi', when='+mpi')
+    # https://github.com/gsjaardema/seacas/blob/master/NetCDF-Mapping.md
+    depends_on('netcdf~mpi maxdims=65536 maxvars=524288')
+    depends_on('hdf5+shared~mpi')
```

Comment:
Shouldn't this depend on `netcdf+mpi` and `hdf5+mpi` when built with MPI parallelism?